colibre/auto_plotter/snia_rates.yml

```diff
@@ -378,22 +421,23 @@ star_formation_rates_snia_rates_50:
       units: "Solar_Mass/year"
   metadata:
     title: SFR-SNIa rate relation (50 kpc aperture)
-    caption: Uses a 50 kpc 3D aperture.
-    section: SNIa Rate
+    caption: Uses a 50 kpc 3D aperture, only select galaxies with a stellar mass above 1e10 Msun.
```

Comment:
Note that the comment says that the mask is > 10^10 Msun, whereas in fact the mask applied is above 10^10.5 Msun. One of the two should be a typo.

Also please use latex format in the caption (i.e. $10^{10} \mathrm{M_\odot}$ instead of 1e10 Msun)